### ISSUE TITLE
Unclear what force-field is for what paper

### ISSUE THREAD

**Issue by jandom** [bestlab/force_fields#1]
There is quite a variety of directories here, it may not be clear to an outside user what force-field is what. 

I was just talking about the water correction paper to a friend (http://pubs.acs.org/doi/abs/10.1021/ct500569b) and without going into the paper I couldn't figure out which directories are the ones I was looking for. Being an insider to the lab, I think it was the ones ending with "ws"

Basically, it would be good to update README.md with what directory is mapped to what paper, so that an outside person can have an easier time using those parameters.

**Comment by bestrob** [comment]
Am in the process of doing this...
